### ISSUE TITLE
The avatr is not speaking

### ISSUE THREAD

**Issue by kofm77** [bornfree/talking_avatar_backend#2]
When I enter speak, a 416 error raises.

What should I do, please?

**Comment by Harikarandev** [comment]
Yes, same issue when I click on speak I can see that the mp3 file is creating but it does not work . Any solution ??

**Comment by Vasu1712** [comment]
@kofm77 @Harikarandev @bornfree I'm also facing same issue, any update on this?

**Comment by Mammouth1979** [comment]
Have you put the Azure keys in the .env file? If so, which Azure service are these keys for? Text to Speech?
